File: Java/Car.java
class Car {
    Integer id;
    String license;
    Account driver;
    Integer passenger;

    public Car(String license, Account driver){
    /*Estas variables son locales al metodo car*/ 
        this.license = license;
        this.driver = driver;
        

    }
    
    void printDataCar(){
        System.out.println("License: " + license + " Name Driver: " + driver.name + " Passenger: " + passenger + " Email: " + driver.email);
    }
        
}

File: PHP/car.php
<?php
    require_once('account.php');
    class Car{
        public $id = integer;
        public $driver = string;
        public $license = string;
        public $passenger = integer;

        public function __construct($license,$driver){
            $this->license = $license;
            $this->driver = $driver;

        }
        public function printDataCar() {
            echo "Licencia: $this->license Driver: ".$this->driver->name;
        }

        public function getPassenger(){
            return $this->passenger;
        }
    }

?>